Project/DataAccess/Context/UnitOfWork.cs

```diff
@@ -2,12 +2,33 @@
 
 namespace DataAccess.Context
 {
-    public sealed class UnitOfWork : System.IDisposable
+    public sealed class UnitOfWork
     {
         // FIELDS
-        private bool disposedValue = false; // To detect redundant calls
         private AppContext context;
+        private static UnitOfWork instance;
+
+        // CONSTRUCTORS

```

Comment:
Maybe `CONSTRUCTORS DESTRUCTOR`?

Reply:
It is conditionally. By this comment I ment thet the block with all method thet are responsible for building and destruction begin.

I did not implement this class, just make an structure. Someone who will work on it could change the comment.